pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1188,6 +1190,15 @@ class Timedelta(_Timedelta):
             if other.dtype.kind == 'm':
                 # also timedelta-like
                 return _broadcast_floordiv_td64(self.value, other, _rfloordiv)
+            elif other.dtype.kind == 'i':
+                # Backwards compatibility
+                # GH-19761
+                msg = textwrap.dedent("""\
```

Comment:
you don't need the textwrap dedent if you put the string over multiple lines with implicit line continuation and string concatentation? 

```
msg = (" ... "
       " ... ")
```

(I would personally find that a bit cleaner (also don't need to `\`), but no big deal)